src/test/ui/span/typo-suggestion.stderr

```diff
@@ -1,14 +1,16 @@
-error[E0425]: unresolved name `bar`
+error[E0425]: unresolved value `bar`
   --> $DIR/typo-suggestion.rs:15:26
    |
 15 |     println!("Hello {}", bar);
-   |                          ^^^ unresolved name
+   |                          ^^^ no resolution found
 
-error[E0425]: unresolved name `fob`
+error[E0425]: unresolved value `fob`
   --> $DIR/typo-suggestion.rs:18:26
    |
 18 |     println!("Hello {}", fob);
-   |                          ^^^ did you mean `foo`?
+   |                          ^^^ no resolution found
```

Comment:
Same as comment above. We're trying to move to using "did you mean `foo`?" as the label text rather than the help because it's less overall text and a little friendlier.